ci/environment.yml

```diff
@@ -21,6 +21,7 @@ dependencies:
   - sqlalchemy-utils
   - sqlalchemy>=1.4.21
   - xarray
+  - xcollection
```

Comment:
@mgrover1, the `to_zarr` and `open_collection` functionality are only available from the main branch. We need to install from git for the time being for everything to work properly